.github/CODEOWNERS

```diff
@@ -949,7 +949,7 @@
 /types/cloud-env/                                         @Morfent
 /types/cloudflare/                                        @Xenfo
 /types/cloudflare-apps/                                   @MartynasZilinskas
-/types/cloudflare-turnstile/                              @jasongerbes
+/types/cloudflare-turnstile/                              @jasongerbes @lbarthon @worenga @olipayne @punkeel @migueldemoura
```

Comment:
isn't that file updated by the bot? (https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#definition-owners)

Reply:
oops